client/src/features/neighborhoods/queries.ts

```diff
@@ -0,0 +1,24 @@
+// @TODO: Move this to .env or something
+const API_BASE_URL = "http://127.0.0.1:4010/api/v1/";
+
+function get(...path: (string | number)[]) {
+  return async function () {
+    const res = await fetch(API_BASE_URL + path.join("/"));
+    if (res.ok) {
+      return await res.json();
+    }
+    throw new Error(`${res.statusText} (${res.status})`);
+  };
+}
```

Comment:
ReactQuery expects either a Promise that resolves to the data or an error to be thrown (which it then parses into `isError`, etc.).